python/lsst/dax/imgserv/getimage/skymapImage.py

```diff
@@ -164,9 +175,14 @@ def get(self, center_coord, width, height, filt, units):
             self._log.debug("begin_y={} end_y={}".format(begin_y, end_y))
             self._log.debug("new_height{} = sBBox.EndY{} - sBBox.BeginY{}".format(
                 new_height, src_exposure.getBBox().getEndY(), expo_bbox.getBeginY()))
-            dest_img[0:new_width, 0:new_height] = src_image[begin_x:end_x, begin_y:end_y]
+            if isinstance(src_exposure, afw_image.Exposure):
```

Comment:
If src_exposure isn't an afw_image.Exposure, then you set dest_img to something, but then you still add dest_exposure to dest_exposure_list.  Seems like you'd only want to append something to that list inside of the if statement?  (Move :183 after :179?)